config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
+  get 'guests/show'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "guests/:id", to: "guests#show"
+
+  post "guests/:id/add_room", to: "guests#add_room"
```

Comment:
This is close and you made it work, however this is not a RESTful route, it doesn't follow the restful conventions. I would have liked this to be either `post "/bookings", to: "bookgins#create"` or could have done `post "guests/:id/bookings, to: "bookings#create"`, but we want to make sure our urls and controller actions follow RESTful conventions.